src/metadata-parser.h

```diff
@@ -113,13 +95,10 @@ namespace librealsense
     class md_time_of_arrival_parser : public md_attribute_parser_base
     {
     public:
-        rs2_metadata_type get(const frame& frm) const override
-        {
-            return (rs2_metadata_type)frm.get_frame_system_time();
-        }
-
-        bool supports(const frame& frm) const override
+        bool find( const frame & frm, rs2_metadata_type * p_value ) const override
         {
+            if( p_value )
+                *p_value = (rs2_metadata_type)frm.get_frame_system_time();
             return true;
```

Comment:
Again we return true on a nullptr address

Reply:
Same thing. Talk to me.